inference-engine/thirdparty/clDNN/src/include/program_node.h

```diff
@@ -293,6 +316,27 @@ struct program_node {
     const std::vector<fused_primitive_desc>& get_fused_primitives() const { return fused_prims; }
     std::vector<fused_primitive_desc>& get_fused_primitives() { return fused_prims; }
 
+#ifdef ENABLE_ONEDNN_FOR_GPU
+    void add_onednn_fused_primitives(std::vector<fused_primitive_desc_onednn> descs) {
+        fused_prims_onednn.erase(fused_prims_onednn.begin(), fused_prims_onednn.end());
+        fused_prims_onednn.insert(fused_prims_onednn.end(), descs.begin(), descs.end());
+    }
+
+    void add_onednn_attrs(std::shared_ptr<dnnl::primitive_attr> attrs) {
+        onednn_attrs = attrs;
+    }
+
+    bool has_out_scales(const std::shared_ptr<dnnl::primitive_attr>& attr);
+    dnnl::post_ops try_optimize_post_ops(dnnl::post_ops& p_ops, const std::shared_ptr<dnnl::primitive_attr>& attr, bool& optimization_is_completed);
+    void set_onednn_primitive_attributes();
```

Comment:
maybe `init_onednn_primitive_attributes` ? `set_*` looks like setter method which may be confusing